### PR TITLE
Fix FDB build

### PR DESCRIPTION
**Pull request by tiagobento**

### CODE REVIEW

File: .github/workflows/ci_full_downstream_build.yml [hunk 1]
```diff
@@ -115,138 +102,37 @@ jobs:
...
+          KOGITO_TOOLING_BUILD_test: "true"
+          KOGITO_TOOLING_BUILD_testIT: "true"
+          KOGITO_TOOLING_BUILD_docker: "true"
+          KOGITO_TOOLING_BUILD_examples: "true"
           DISPLAY: ":99.0"
+          START_SERVER_AND_TEST_INSECURE: "true"
         run: |
-          cd ${{ github.workspace }}/kogito-tooling
-          yarn bootstrap && yarn build:prod
+          lerna run build:prod --stream --concurrency 1
```
Comment (domhanak) [inline .github/workflows/ci_full_downstream_build.yml:123]: so no `yarn bootstrap` before we begin building? Is that expected?

Comment (tiagobento) [reply .github/workflows/ci_full_downstream_build.yml:123]: It's part of the `setup-env` action on `kogito-tooling`.

**Comment by tiagobento** [comment]
Merging as we need to do it to test.

File: .github/workflows/ci_full_downstream_build.yml (hunk 1 above)
Comment (domhanak) [reply .github/workflows/ci_full_downstream_build.yml:123]: How is that action run on `kogito-editors-java` ? if it is a action of `kogito-tooling` am I missing something?

Comment (domhanak) [reply .github/workflows/ci_full_downstream_build.yml:123]: But imho this part will be missing `cd ${{ github.workspace }}/kogito-tooling` the error in #114 suggests that

**Pull request merged**